src/phew/server.py

```diff
@@ -266,7 +268,9 @@ async def _handle_request(reader, writer):
     if request.headers["content-type"].startswith("multipart/form-data"):
       request.form = await _parse_form_data(reader, request.headers)
     if request.headers["content-type"].startswith("application/json"):
-      request.data = await _parse_json_body(reader, request.headers)
+      raw_body, parsed_data = await _parse_json_body(reader, request.headers)
+      request.raw_data = raw_body
+      request.data = parsed_data
     if request.headers["content-type"].startswith("application/x-www-form-urlencoded"):
```

Comment:
I've modified this part of phew because I sometimes need the raw json string (mostly during authentication)